src/styles/global.scss

```diff
@@ -319,3 +319,11 @@ hr {
     }
   }
 }
+
+.blink {
+  transition: opacity 0.5s ease-in-out;
+
+  &.hide {
+    opacity: 0;
+  }
+}
```

Comment:
In my opinion it would be better to create CSS animation and then on the component level adding only one class, e.g.: `value-changed` this way:

```
watch(<modal value>, async () => {
  valueChanged.value = true;
  await nextTick();
  valueChanged.value = false;
});
```

The style for this class would be:
```
.value-changed {
  animation: blink-animation...
}
```

(just a rough idea, not tested)